model-optimizer/extensions/front/onnx/resize_ext.py

```diff
@@ -1,12 +1,9 @@
 """
  Copyright (C) 2018-2020 Intel Corporation
-
```

Comment:
Restore removed lines in the copyright. They must be there.

Reply:
Fixed.